Makefile

```diff
@@ -0,0 +1,14 @@
+NAME    := slurm
+SRC_EXT := bz2
+VERSION := 18.08.8
+URL_BASE :=  https://download.schedmd.com/$(NAME)
+SOURCE   = $(URL_BASE)/$(NAME)-$(VERSION).tar.$(SRC_EXT)
+
+spec: $(NAME).spec Makefile
```

Comment:
This shouldn't be needed.  Dependencies in `Makefile_packaging.mk` should be taking care of this.

Reply:
I was not finding a way to make sure that a new tarball was downloaded only if either the Makefile or spec file changed.

The existing Makefile_packaging.mk only re-downloads the tarball if it is missing via one of several rules.

Reply:
Why does it need to be re-downloaded if it's already downloaded?  Not recreating a target which is newer than it's dependencies is `make` doing what it does best.

Reply:
Should have been a wild-card removal of the tarball, to clean up older tarballs when there is a version change.

Reply:
Changing to a rule for cleaning up obsolete tarballs when the version changes.